ecosystem/sep-0036.md

```diff
@@ -0,0 +1,137 @@
+
+This SEP is based on [SEP-0008](sep-0008.md), but the issuer no long has to a co-signer on the account holding the regulated asset. It also updates the API of the approval server to better align with other active SEPs.
+
+**Target Audience**: Asset issuers, issuance platforms, wallet developers, and exchanges
+
+## Motivation
+
+Stellar aims to be an ideal platform for issuing securities. As such, it must provide tools to comply with various regulatory requirements. Regulation often requires asset issuers to monitor and approve every transaction involving the issued asset and to enforce an assortment of constraints. This SEP provides support for these requirements.
+
+## Overview
+
+Implementing a Regulated Asset consists of these parts:
+- [Stellar.toml]: Issuers will advertise the existence of an Approval Service and the approval criteria via their stellar.toml file.
+- [Approval Server]: HTTP protocol for transaction signing.
+- [Transaction Composition]: Wallets will have to put operations that fully authorize the accounts for the asset, transact the asset, and deauthorize the accounts in a single transaction.
+
+## Regulated Assets Approval Flow
+
+1. User creates and signs a transaction.
+2. Wallet resolves asset information and detects that it's a regulated asset.
+3. Wallet sends the transaction to the approval server.
+4. Approval server determines whether the transaction is compliant based on the compliance ruleset and looks up the current state of the ledger if necessary.
+5. Wallet handles approval response:
+    1. *Success?* Transaction has been approved and signed by issuer.
+    2. *Rejected?* Wallet should display given error message to the user.
+
+## Stellar.toml
+
+Issuers will advertise the existence of an Approval Service through their stellar.toml file. This is done in the [[CURRENCIES]] section as different assets can have different requirements.
+
+### Fields:
+
+- `regulated` is a boolean indicating whether or not this is a regulated asset. If missing, `false` is assumed.
+- `approval_server` is the url of an approval service that signs validated transactions.
+- `approval_criteria` is a human readable string that explains the issuer's requirements for approving transactions.
+
+### Example
+
+```toml
+[[CURRENCIES]]
+code="GOAT"
+issuer="GD5T6IPRNCKFOHQWT264YPKOZAWUMMZOLZBJ6BNQMUGPWGRLBK3U7ZNP"
+regulated=true
+approval_server="https://goat.io/tx_approve"
+approval_criteria="The goat approval server will ensure that transactions are compliant with NFO regulation"
+```
+
+## Approval Server
+
+The transaction approval server receives a signed transaction and checks for compliance and signs it on success.
+
+### Possible results
+- Success: Transaction was found compliant and signed by service.
+- Rejected: Transaction was rejected.
+
+### Request
```

Comment:
I would take a look at other SEPs and how they define endpoints. Most have a bullet list with links to each endpoint request/response definition. The specific HTTP method and URI are also often used in the title.

Reply:
I think we also need examples for both the JSON and formurlencoded requests and a table for the request parameters like the response ones.

e.g. https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0030.md#endpoints